### PR TITLE
Fix broken test descriptions

### PR DESCRIPTION
**Pull request by smowton**
These two tests accidentally passed one of their required outputs as a command-line argument.

### CODE REVIEW

**Review by tautschnig — CHANGES_REQUESTED** [review]
Acting against all rules on how to be a good reviewer: more work, please.

It's great that you have noticed those problems (I'm actually wondering how), but the reason those tests still passed is that the test descriptions do not include
```
^EXIT=0$
^SIGNAL=0$
```
Being the terrible person that I am: can you please, as part of this PR, add this to all *.desc files that are missing it, with those saying "VERIFICATION FAILED" using `EXIT=10`instead. Thank you!

**Comment by smowton** [comment]
@tautschnig done

File: regression/ansi-c/Forward_Declaration2/test.desc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 CORE
 main.c
 
+^EXIT=1$
```
Comment (tautschnig) [inline regression/ansi-c/Forward_Declaration2/test.desc]: I believe this needs to be `^EXIT=(64|1)$` to cope with Windows.

File: regression/ansi-c/Incomplete_Type1/test.desc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 CORE
 main.c
 
+^EXIT=1$
```
Comment (tautschnig) [inline regression/ansi-c/Incomplete_Type1/test.desc]: I believe this needs to be `^EXIT=(64|1)$` to cope with Windows.

File: regression/ansi-c/static2/test.desc [hunk 3]
```diff
@@ -2,5 +2,7 @@ CORE
 main.c
 main2.c --function foo
 ^main symbol `foo' is ambiguous$
+^EXIT=1$
```
Comment (tautschnig) [inline regression/ansi-c/static2/test.desc]: I believe this needs to be `^EXIT=(64|1)$` to cope with Windows.

File: regression/ansi-c/static3/test.desc [hunk 4]
```diff
@@ -2,5 +2,7 @@ CORE
 main.c
 main2.c --function foo
 ^main symbol `foo' is ambiguous$
+^EXIT=1$
```
Comment (tautschnig) [inline regression/ansi-c/static3/test.desc]: I believe this needs to be `^EXIT=(64|1)$` to cope with Windows.

**Review by tautschnig — COMMENTED** [review]
Thank you very much for all the extra work! Just some requests in ansi-c to cope with a portability problem.

File: regression/ansi-c/static3/test.desc (hunk 4 above)
Comment (smowton) [reply regression/ansi-c/static3/test.desc]: Do you know why?

Comment (smowton) [reply regression/ansi-c/static3/test.desc]: Ah never mind I got it, goto-cc vs. goto-cl

**Comment by smowton** [comment]
@tautschnig changes applied, CI is passing except for CodeBuild which is opaque to me and AFAIK not yet intended to work

**Comment by tautschnig** [comment]
This now needs a rebase. Maybe @kroening can in the meantime explain what's going wrong with CodeBuild?

**Review by tautschnig — APPROVED** [review]

**Comment by smowton** [comment]
@tautschnig done, let's see if CodeBuild complains again

**Comment by tautschnig** [comment]
regression/cbmc-java/lambda1 seems to conflict with symex-lazy-loading, if my diagnosis is correct.

**Comment by smowton** [comment]
Yup, and the new test revealed it. Adding an entry point so the test can work under either mode now.

**Comment by smowton** [comment]
Fixed.

**Pull request merged**